api/pom.xml

```diff
@@ -21,18 +21,25 @@
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-        <groupId>jakarta.enterprise.concurrent</groupId>
-        <artifactId>jakarta.enterprise.concurrent.parent</artifactId>
+        <groupId>jakarta.concurrent</groupId>
+        <artifactId>jakarta.concurrent.parent</artifactId>
         <version>3.1.1-SNAPSHOT</version>
     </parent>
 
-    <artifactId>jakarta.enterprise.concurrent-api</artifactId>
+    <artifactId>jakarta.concurrent-api</artifactId>
+
+    <distributionManagement>
+        <relocation>
```

Comment:
I'm not an expert on `relocation` element, but I'm pretty sure it is to work the other way around.

In NEW version under OLD coordinates pom-type artifact with `relocation` pointing to new GAV is released - example: https://repo.maven.apache.org/maven2/org/slf4j/slf4j-log4j12/2.0.13/slf4j-log4j12-2.0.13.pom.

Reply:
@pzygielo thanks for pointing that out.  I assumed that the relocation would produce something like this: 
![Screenshot 2024-07-09 at 15 25 40](https://github.com/jakartaee/concurrency/assets/33664635/ff47a275-e1d1-4a0e-9937-4245ec912dd3)

Do you have an alternative solution to accomplish this?

Reply:
That's what happens with log4j12:

![image](https://github.com/jakartaee/concurrency/assets/11896137/5222a36b-c26f-4305-8175-368bdeb15d5b)

But note the direction - **relocated to**.

Reply:
> Do you have an alternative solution to accomplish this?

But having `relocation` as here in this PR would suggest the meaning `I am relocated FROM jakarta.enterprise.concurrent:jakarta.enterprise.concurrent-api`. I suppose the clients already using NEW G:A will not be specially interested where it was published before. But the current clients of OLD version will see no upgrades for old G:A, unless under OLD G:A new version with relocation is available. This will not happen just by using `relocation` as proposed.

That's what I think.

Reply:
@pzygielo I was following https://maven.apache.org/guides/mini/guide-relocation.html to get the relocation correct.  Which suggests that you need to have two poms.